custom/enikshay/reports/repeaters.py

```diff
@@ -273,7 +285,7 @@ def _get_voucher_repeater_ids(self):
 
     @property
     def total_records(self):
-        return self._search_results().total
+        return self._search_results(paged=True).total
```

Comment:
That's counterintuitive, I'd have expected `paged=False` to return the full count..

Reply:
yeah, this is a bit confusing. the HQES `total` property returns the count of all results for a specific query, not just the number returned. I'm using the memoized decorator so that it doesn't run the query twice. 

I don't think this gets called when a report is run against all rows. I've removed the argument to make it a bit less confusing while reading this.